src/sth_server.js

```diff
@@ -0,0 +1,159 @@
+/* globals module, process, require */
+
+(function() {
+  "use strict";
+
+  var hapi = require('hapi');
+  var joi = require('joi');
+
+  var server, sthDatabase, sthConfig, sthLogger, sthHelper;
+
+  /**
+   * Starts the server asynchronously
+   * @param {string} host The host where the server will be running
+   * @param {string} port The port where the server will be listening
+   * @param {string} aSTHDatabase The database to be used by the server
+   * @param {Function} callback Callback function to notify the result
+   *  of the operation
+   */
+  function startServer(host, port, aSTHDatabase, callback) {
+    sthDatabase = aSTHDatabase;
+
+    server = new hapi.Server();
+
+    server.connection({
+      host: host,
+      port: port
+    });
+
+    server.route([
+      {
+        method: 'GET',
+        path: '/STH/v1/contextEntities/type/{type}/id/{entityId}/attributes/{attributeId}',
```

Comment:
Yeah, it follows the defined API. I won't post it here but please contact me for further details ;)